doc/news/_preparation_next_release.md

```diff
@@ -120,7 +122,7 @@ The text below summarizes updates to the [C (and C++)-based libraries](https://w
 
 ### Core
 
-- <<TODO>>
+- Remove `keyRewindMeta`, `keyCurrentMeta`, `ksHead`, and `ksTail` functions for internal iteration of `Keyset`s and Metadata of `Key`s _(Florian Lindner @flo91)_
```

Comment:
Please, also mention this in the "Highlights" section at the top of the file. Also clearly mark it as a breaking change, and refer to the migration guide (`doc/dev/iterators.md`).